notary-api/src/operations/notarize.js

```diff
@@ -0,0 +1,60 @@
+import uuidv4 from 'uuid/v4';
+import { addNotarizationJob } from '../queues/notarizationsQueue';
+import { notarizationResults } from '../utils/stores';
+import { packMessage, sha3 } from '../utils/wibson-lib/cryptography/hashing';
+import { packPayData } from '../blockchain/batPay';
+import { getAccount } from '../services/signingService';
+
+const createNotarization = async ({
```

Comment:
This function does not await, so it isn't async.

Reply:
I'm awaiting for `getAccount()` and now for the `store()`